docs/pages/guides/using-sentry.mdx

```diff
@@ -47,7 +47,7 @@ them later:
 
 <Step label="1.2">
 
-Go to the [Developer Settings > Auth Tokens](https://sentry.io/settings/) page and create a new token. The token is automatically scoped for Source Map Upload and Release Creation. Save it.
+Go to the [Developer Settings > Auth Tokens](https://sentry.io/settings/auth-tokens/) page and create a new (organization, not user) token. The token is automatically scoped for Source Map Upload and Release Creation. Save it.
```

Comment:
```suggestion
Go to the [Developer Settings > Auth Tokens](https://sentry.io/settings/auth-tokens/) page and create a new Auth Organization token. The token is automatically scoped for Source Map Upload and Release Creation. Save it.
```

The **Auth Tokens** tab inside the Sentry dashboard only allows to create auth token for Organization. I think we can explicitly mention that here since User Auth tokens are only available under User > User Auth Tokens settings.